.prettierrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": false,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "arrowParens": "avoid",
+  "proseWrap": "preserve",
+  "semi": false,
+  "endOfLine": "lf"
+}
```

Comment:
@Whisker17 ping :)

Reply:
As I said before, my original plan was to prettier format the entire grant site. Since the website uses Docusaurus, there may be some `.jsx` or `.tsx` files, not just for `.md`.

In addition, I have also noticed the possible disadvantages of using husky hooks, so I am thinking about whether to close this PR. If the grant site won't have website design expansion in the future (just the modification and addition of md files), I think the comments [here](https://github.com/w3f/Grants-Program/pull/1863#pullrequestreview-1572145074) are right, and you can choose to close this PR if you agree too.